File: CRM/src/app/shared/eHospital_DongNai_A/CDT_TrangThietBiTuyenDuoi.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CDT_TrangThietBiTuyenDuoi } from './CDT_TrangThietBiTuyenDuoi.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CDT_TrangThietBiTuyenDuoiService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'CDT_TrangThietBi_Id', 'ThietBi_Id', 'BenhVien_Id', 'TinhTrang_Id', 'NgayQuanLy', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id', 'GhiChu'];

    List: CDT_TrangThietBiTuyenDuoi[] | undefined;
    ListFilter: CDT_TrangThietBiTuyenDuoi[] | undefined;
    FormData!: CDT_TrangThietBiTuyenDuoi;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CDT_TrangThietBiTuyenDuoi";
    }
}

File: Service_eHospital_DongNai_A/Implement/CDT_TrangThietBiTuyenDuoiService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class CDT_TrangThietBiTuyenDuoiService : BaseService<CDT_TrangThietBiTuyenDuoi, ICDT_TrangThietBiTuyenDuoiRepository>
    , ICDT_TrangThietBiTuyenDuoiService
    {
    private readonly ICDT_TrangThietBiTuyenDuoiRepository _CDT_TrangThietBiTuyenDuoiRepository;
    public CDT_TrangThietBiTuyenDuoiService(ICDT_TrangThietBiTuyenDuoiRepository CDT_TrangThietBiTuyenDuoiRepository) : base(CDT_TrangThietBiTuyenDuoiRepository)
    {
    _CDT_TrangThietBiTuyenDuoiRepository = CDT_TrangThietBiTuyenDuoiRepository;
    }
    }
    }

